Release v0.6.0?

bdero opened the pull request:
This is going off of the latest semantic-looking version here: https://github.com/starteam/starcellbio_html/releases/tag/v0.5

There are several release tags that are dates, like `V20140925`, for instance. Some releases follow another pattern as well.
We should probably discuss and think about versioning, and perhaps a roadmap for StarCellBio moving forward.

This probably shouldn't be merged until we have a clean production environment ready to go.

skini committed:
made changes for focus and blur and if change

skini committed:
info link change and adjusting films bands

skini committed:
spacing in css files

skini committed:
fixed copy error in experiment setup

skini committed:
all dev changes from revised half list

skini committed:
changes from alison, need to address bug from lourdes

skini committed:
fixed a microscopy error with slides

skini committed:
trying something

skini committed:
trying something, and it worked

skini committed:
trying something, with images

skini committed:
trying something, with images

skini committed:
fixed centering

skini committed:
a few changes

skini committed:
watch.py modification and fixing some errors

skini committed:
finished up dev page changes

skini committed:
working on new problem set

skini committed:
add all of the problem sets back

skini committed:
fix issues with westernblot

skini committed:
fixed cyclin B, no autocomplete for course code, and redid index

skini committed:
changes to captalization

skini committed:
changes to graphs

skini committed:
update with graphs, new equations and western blotting techniques

skini committed:
reset timestamp

skini committed:
fixed wb, not facs, parsers

skini committed:
make normal and high levels more pronounced

skini committed:
new changes to the problem set

skini committed:
new changes to the problem set

skini committed:
master changes model

skini committed:
finished the problem set 2, with randomization and everything else

skini committed:
new round changes

skini committed:
fixed small changes, microscopy navigation

skini committed:
fixed microscope navigation bug

skini committed:
starting accessibility with homepage experiment design

skini committed:
more accessibility, first pass withouth techniques

skini committed:
more accessibility changes, starting the techniques

skini committed:
change to remove all errors

skini committed:
more minor changes, not started techniques accessibility

skini committed:
update microscopy images

skini committed:
fix black text

skini committed:
add facs graph revised

skini committed:
update westernblot version

skini committed:
fix siRNA5

skini committed:
changes again with microscopy and westernblot

skini committed:
fixed band levels rna2

skini committed:
add pictures to unequal

skini committed:
final changes to problem set

skini committed:
move western blot gel higher, and mad2 as always last option

skini committed:
made crucial changes

skini committed:
fixed images in folders

skini committed:
fix token bug from lourdes, server save

skini committed:
new instructions

skini committed:
rounded data

skini committed:
fixed print icon test

skini committed:
have started working on accessibility for techniques

skini committed:
finished western_blot, need to go back and check, still need to finish facs and start microscopy

skini committed:
finished all techniques, need to test the techniques

skini committed:
redoing selenium tests

skini committed:
new tests fixed up to line 144

skini committed:
added few more tests

skini committed:
starting FACS, need to go back and do signin and userguide, for testing

skini committed:
did basic facs test

skini committed:
added button for double range

skini committed:
facs create double_analysis

skini committed:
new bisector temporary icon

skini committed:
built primitive bisector gates and added setup for FACS

skini committed:
working on the double gates, fixed the selection

skini committed:
fixed sample prep error for FACS, and fixed issues with moving middle marker for FACS bisector gate

skini committed:
comments for mastermodel randomize

skini committed:
created second problem set to demo FACS features

skini committed:
starting microscopy test in selenium

skini committed:
add updated tests for microscopy, up to setup

skini committed:
more testing stuff

skini committed:
clean up code, removed magic numbers and some tag references, fixed facs side bar length, and cleaned up MicroscopyView

skini committed:
starting facs changes, cleaned up microscopy code more, fixed list[0] problem

skini committed:
fixed problem set

skini committed:
merge accessibility back into main branch

skini committed:
fixed merge mastermodel

skini committed:
fix a few facs issues, and remove microscopy error temporarily

skini committed:
add all microscopy changes, save few

skini committed:
fix libraries in html

skini committed:
fixed problem set microscopy usability testing

skini committed:
third round of changes microscopyusability test

skini committed:
changes for server

skini committed:
fix model

skini committed:
fix text on load slides note

skini committed:
starting changes to facs gates, new bisector, first version dots

skini committed:
FACS capability changes, with dots, new interaction, flowJo simulation

skini committed:
fixed errors for facs

skini committed:
change text on ? button

skini committed:
facs changes finish up

skini committed:
fixed small issues

skini committed:
small bug fixes FACS

skini committed:
fix small error with the drag click of the bisector gate from usability testing

skini committed:
starting notebook, built template and parser, need to fix between tabs location

skini committed:
starting to create a single section

skini committed:
starting ui elements

skini committed:
able to add text nodes to notebook, started ui, figured out parser for images, need to build image templates, and reorderability

skini committed:
finished western blot for adding, finished facs for adding, and experiment design and setup, in the middle of microscopy for adding

skini committed:
high priority bugs from microscopy usability testing, save one

skini committed:
able to add one microscopy slide to section

skini committed:
addressed all high priority microscopy usability bugs

skini committed:
fixed microscopy and facs_lane and added notebook element class to create notebook

skini committed:
revisions to high priority bugs from microscopy usability test

skini committed:
starting to add the element to every notebookelement parsing for multiple experiments

skini committed:
microscopy usability changes revisions, pass 3

skini committed:
starting fixes for notebook parsing

skini committed:
microscopy usability third round changes, save for scroll change

skini committed:
add error message

skini committed:
add error message

skini committed:
add error message

skini committed:
add error message

skini committed:
add error message

skini committed:
add error message

skini committed:
add error message

skini committed:
add error message

skini committed:
starting instructor account setup

skini committed:
microscopy usability fifth round changes

skini committed:
starting instructor accounts

skini committed:
starting to create instructor account requests

skini committed:
creating options for instructor/student return results

skini committed:
instructor backend request complete, edits still need to be coded, working on front end UI

skini committed:
working on dashboard and experiment setup, able to create new assignment

skini committed:
working on experiment setup, revisions to include archive, working on dashboard interface

skini committed:
working on experiment setup, changed dashboard UI, changed course UI, working on autosave

skini committed:
change delete

skini committed:
experiment setup refactored, urls refactored, on first page of experiment setup from refactor, working on empty model

skini committed:
working on experiment setup page 2, finished the strains and treatment selection, working on the treatment specifications and matrix

skini committed:
halfway through page 2 of experiment setup

skini committed:
finished second page of experimental setup, working on sorting and selecting rows for final setup

skini committed:
starting western blot

skini committed:
merge with microscopy usability changes

skini committed:
created base files for all screens, working on wb3

skini committed:
remove deleted files

Commit:
added monitor file

skini committed:
halfway through western blot, made rudimentary navigation

skini committed:
merge with lab notebook

skini committed:
merge with lab notebook

skini committed:
merge notebook again

skini committed:
finished first version table for western blot

skini committed:
last commit, finished rudimentary western_blot

iceraj committed:
Syntax Error fixes

iceraj committed:
Minor Changes/Updates to Make to StarCellBio - HOMEPAGE

iceraj committed:
Watch to PROJECT_HOME vs VIRTUAL_ENV

iceraj committed:
Fix the positioning of the "1. Design" button in the navigation bar

iceraj committed:
Remove the 'In the Lab' feature box

iceraj committed:
Change the name from 'Lab Notebook' to 'Lab Report'

iceraj committed:
Merge branch 'feature/minor_changes' into develop

iceraj committed:
Format file

iceraj committed:
Remove zip-js

iceraj committed:
Remove SCB_REST

iceraj committed:
Remove build and scbU folders.

iceraj committed:
Remove extra files

iceraj committed:
Remove Slides-SlidesJS from repo

iceraj committed:
Remove microscopy folder

iceraj committed:
Remove build folder

iceraj committed:
Working on SCB_EX1

iceraj committed:
Working on Ex1 SB1

iceraj committed:
Add views changest

iceraj committed:
Finising first draft of SCB_EX1

iceraj committed:
Evernote: ex1 Assignments page

iceraj committed:
Ex1 fixes

iceraj committed:
StarCellBio - working on Template

carsongee committed:
Add vagrantfile and ansible play for creating dev environment

(cherry picked from commit b5fe4f5badbd46086efc84017d7af26bbdec9e49)

Conflicts:
	.gitignore
	docs/DeveloperGuide.rst
	requirements.txt

skini committed:
all of Anna changes plus first draft of Singapore assignment

annagav committed:
mod files

carsongee committed:
Delete unneccessary files

annagav committed:
Merge pull request #15 from carsongee/feature/remove_built_files

Delete unneccessary files

skini committed:
minor changes, checked exercises 1 and 2, started exercise 3

carsongee committed:
Move MacFSEvents to separate requirements file since it requires Mac

carsongee committed:
Correct blind indexing
Allow watch to be run on non-Mac systems
Add PROJECT_HOME to environment
Add step to compile assets in provision

carsongee committed:
Add environment variable definition to watch.py step

annagav committed:
fixed syntax errors in master_model.js

bdero committed:
Fix outdated fixtures to match the models

bdero committed:
Merge pull request #18 from starteam/bdero/fix_fixtures

Fix outdated fixtures to match the models

bdero committed:
Merge pull request #19 from starteam/feature/add_vagrant

Development Environment

bdero committed:
Merge pull request #16 from starteam/skini/exercises_15

minor changes, checked exercises 1 and 2, started exercise 3

annagav committed:
addedd instructions to __scb_sample_3

pdpinch committed:
Merge pull request #20 from annagav/feature/super_safe

Exercise 3 text: background information

annagav committed:
fixed setup_experiment step of ex3

annagav committed:
modified Vagrantfile and micro_kinds

bdero committed:
Port forward 8000 with vagrant.

This is useful when private networks don't work.

bdero committed:
Rename images to match production deployment

bdero committed:
Remove all generated .soy.js files

bdero committed:
Remove all generated .gss.css files

bdero committed:
Ignore all generated files

carsongee committed:
Merge pull request #26 from bdero/feature/bdero/fix-image-names

Fix image names to match production deployment

annagav committed:
a stub for microscopy parser

bdero committed:
Configuration overrides

bdero committed:
Ignore yaml configuration

bdero committed:
Database settings easily overridable

bdero committed:
Remove commented code & pyflakes errors

bdero committed:
Ignore gen directories instead

bdero committed:
Merge pull request #28 from bdero/feature/bdero/static-cleanup

Remove and ignore all generated .soy.js and .gss.css files

bdero committed:
Use map and filter instead of list comp

bdero committed:
Merge pull request #29 from bdero/feature/bdero/layered-configuration

Settings configurable with untracked config and environment variables

bdero committed:
Extract developer guide from #14

bdero committed:
Port forward 8100 with vagrant instead

bdero committed:
Use watchdog as the watcher.

- Cleaned up watch.py quite a bit
- Add watchdog observer in
- Use docopt for argument parsing
- Use python logger for the watcher

carsongee committed:
Merge pull request #27 from bdero/feature/bdero/vagrant-port-forward

Port forward 8100 with vagrant.

annagav committed:
minor changes to select_slide_type

annagav committed:
created SUTD_assignment 2 folder

annagav committed:
fixed a bug

carsongee committed:
Merge pull request #31 from bdero/feature/bdero/watchdog

Use watchdog for static file watching

annagav committed:
finished western blot, facs in progress

annagav committed:
facs done

carsongee commented:
Totally un-reviewable, plus merge conflicts, but hey I'm :+1:  for 0.6.0 :hammer: . Can you maybe point to where that version is specified in the code?

annagav committed:
completed microscopy parser and started westernblot parser

annagav committed:
modified microscopy IF select conditions

annagav committed:
fixed assignment setup and add_sample_dialog

pdpinch committed:
Merge pull request #45 from annagav/feature/SUTD_assignment1

fixed assignment setup and add_sample_dialog; fixes #37

annagav committed:
added microscopy to SUTD assignment 2

annagav committed:
creted new unequal_segregation graph, changed scale

pdpinch committed:
Merge pull request #35 from annagav/feature/exercise3

Feature/exercise3

Setup Experiment: Fixed treatment_id and 30 sec treatment duration.
Microscopy: In progress.
FACS: untouched.
Western Blot: untouched.

annagav committed:
fix data normalization in FACSModelFactory

pdpinch committed:
Merge pull request #32 from annagav/feature/sutd_assignment2

Feature/sutd assignment2

annagav committed:
Fixes instruction page for the assignment

- Disabled arrow buttons if there is only one instruction.
- Changed the instruction text of the sutd2 assignment.

Please enter the commit message for your changes. Lines starting

annagav committed:
fixed microscopy image overlay

pdpinch committed:
Merge pull request #64 from annagav/feature/microscopy_img_overlay_56

fixed microscopy image overlay: fixes #56

annagav committed:
sutd2: added select all button

annagav committed:
disbled LAB REPORT button

annagav committed:
sutd2: added more images to slide_parser

annagav committed:
sutd2: fixed close add sample dialog button

annagav committed:
facs: choose condition

annagav committed:
sutd2: fix antibody name

pdpinch committed:
Merge pull request #83 from annagav/feature/sutd2_western_blot_antibody_69

sutd2: fix antibody name. Fixes #69 and #70

pdpinch committed:
Merge pull request #82 from annagav/feature/sutd2_close_add_samples_68

sutd2: fixed close add sample dialog button. Fixes #68

carsongee committed:
Merge pull request #30 from bdero/feature/bdero/developer_guide

Extract developer guide from #14

carsongee committed:
Switch contact form to use form validation and django e-mail
Still defaults to star@mit.edu, but is configurable via django settings
Closes #84

pdpinch committed:
Merge pull request #67 from annagav/feature/sutd2_select_all_btn

sutd2: added select all button: Fixes #66

pdpinch committed:
resizing sutd2 micrscopy images to 400 px high. fixes #77

annagav committed:
delete unnecessary lines

pdpinch committed:
Merge pull request #63 from annagav/feature/instruction_page_fixes_58

Fixes #58 instruction page for the assignment

annagav committed:
Merge pull request #86 from starteam/feature/resize-sutd2-microscopy

resizing sutd2 micrscopy images to 400 px high. fixes #77

annagav committed:
gray out the LAB REPORT button

bdero committed:
Merge pull request #85 from starteam/feature/cg/contact_us

Switch contact form to use form validation and django e-mail

pdpinch committed:
Merge pull request #81 from annagav/feature/sutd2_facs_blank_conditions_72

FACS: choose condition. Fixes #72

pdpinch committed:
Merge pull request #79 from annagav/feature/sutd2_add_images_to_slide_parser

sutd2: added more images to slide_parser

annagav committed:
SUTD: changed assignment names

annagav committed:
FACS: fix green button misalignment. fixes #71

annagav committed:
removed LAB REPORT button

carsongee committed:
Optimize asset pipeline for CSS and Javascript
Create deterministic jasvascript loading
Minify all JS into single file with ``build.py -p``
Minify all CSS into single file with ``build.py -p``

bdero committed:
Merge pull request #91 from starteam/feature/cg/hardcode_javascript_deps

Optimize asset pipeline for CSS and Javascript

pdpinch committed:
Merge pull request #89 from annagav/feature/change_assignment_names

SUTD: changed assignment names

pdpinch committed:
Merge pull request #90 from annagav/feature/facs_green_buttons

FACS: fix green button misalignment. fixes #71

pdpinch committed:
Merge pull request #80 from annagav/feature/disable_lab_report_btn

disbled LAB REPORT button: fixes #57

annagav committed:
facs: fix top offset. fixes #98

pdpinch committed:
Merge pull request #99 from annagav/feature/facs_green_buttons

facs: fix top offset. fixes #98

annagav committed:
fix xaxis start point in FACSModelFactory

pdpinch committed:
Merge pull request #100 from annagav/feature/facs_gates_73

fix xaxis start point in FACSModelFactory: fixes #73

carsongee committed:
Add CSS to yaml defined assets

annagav committed:
Merge pull request #101 from starteam/feature/css_ordering

Add CSS to yaml defined assets

bdero commented:
Using release branch instead.. Ditching master.
